code/stable/dblpendulum/SRS/HTML/DblPendulum_SRS.html

```diff
@@ -371,6 +371,10 @@ <h2>Abbreviations and Acronyms</h2>
                   <td>DD</td>
                   <td>Data Definition</td>
                 </tr>
+                <tr>
+                  <td>DblPend</td>
+                  <td>DblPendulum</td>
```

Comment:
Shouldn't the full version of this be "Double Pendulum"?

Reply:
In that case should the abbreviation be "DblPend" or "DblPendulum"

Reply:
I like DblPend, since it is shorter.